assets/pages/bundles-landing/components/subscribeNewDesign.jsx

```diff
@@ -0,0 +1,49 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+import InfoSection from 'components/infoSection/infoSection';
+
+import SubscriptionBundle from './subscriptionBundleNewDesign';
+
+
+// ----- Component ----- //
+
+export default function Subscribe() {
+
+  return (
+    <div className="subscribe-new-design">
+      <InfoSection heading="subscribe" className="subscribe-new-design__content gu-content-margin">
+        <div className="subscribe-new-design__bundles-wrapper">
+          <SubscriptionBundle
+            heading="digital"
+            price="11.99"
+            from={false}
+            copy="Get our journalism across up to 10 devices, to enjoy wherever you go"
+            ctaText="Start 14-day trial"
+            image="7c7b9580924281914e82dc163bf716ede52daa8b/0_0_600_360"
```

Comment:
I wonder if it is possible to create meaningful keys for this ids and keep them all together in a file. For example, imageMeaningfulPartHere

Reply:
I'm inclined to do that if we end up using the same images in multiple pages. For the moment, since they're just used the once, I think it's simpler if they're inline?

Reply:
Actually, after discussion offline, yeah this is a good idea, will do.

Reply:
Done.